src/linagora.esn.controlcenter/app/app.run.js

```diff
@@ -1,12 +1,11 @@
-(function(angular) {
-  'use strict';
+angular.module('linagora.esn.controlcenter')
 
-  angular.module('linagora.esn.controlcenter')
+  .run(function(dynamicDirectiveService, DynamicDirective) {
+    var profileMenuControlcenter = new DynamicDirective(true, 'profile-menu-controlcenter');
 
-    .run(function(dynamicDirectiveService, DynamicDirective) {
-      var profileMenuControlcenter = new DynamicDirective(true, 'profile-menu-controlcenter');
+    dynamicDirectiveService.addInjection('profile-menu-controlcenter', profileMenuControlcenter);
+  }).run(fillTemplates);
```

Comment:
Break the new run block to a new line

```suggestion
  })
  .run(cacheTemplates);
```